backend/mora/service/employee.py

```diff
@@ -49,10 +49,60 @@ class EmployeeDetails(enum.Enum):
     INTEGRATION = 2
 
 
-class EmployeeRequestHandler(handlers.RequestHandler):
+class EmployeeRequestHandler(handlers.ReadingRequestHandler):
     __slots__ = ('details_requests',)
     role_type = "employee"
 
+    @classmethod
+    def has(cls, scope, reg):
+        pass
+
+    @classmethod
+    def get(cls, c, search_fields):
+
+        scope = c.bruger
+
+        object_tuples = list(scope.get_all(
+            **search_fields,
+        ))
+
+        return [
```

Comment:
Jeg er slet ikke tosset med den list-comprehension.

Reply:
Det må jeg erklære mig enig i, en list-comprehension på 19 linjer direkte i return er nok heller ikke lige min kop te.

Reply:
> Det må jeg erklære mig enig i, en list-comprehension på 19 linjer direkte i return er nok heller ikke lige min kop te.

Til Christoffers forsvar er det blot en omorganisering af noget kode jeg skrev for noget tid siden. (Og nej, det er ikke just den mest _letlæselige_ kode jeg har skrevet…)

Reply:
Som nævnt er det bare kode der er flyttet rundt - det er ikke specielt pænt, men at fikse det er out of scope for denne feature synes jeg.